etna/whatson/models.py

```diff
@@ -514,10 +529,19 @@ class EventPage(ArticleTagMixin, TopicalPageMixin, BasePageWithIntro):
     venue_space_name = models.CharField(
         max_length=255,
         verbose_name=_("venue space name"),
+        null=True,
         blank=True,
         help_text=_("The name of the venue space."),
     )
 
+    venue_directions = models.URLField(
+        max_length=255,
+        verbose_name=_("venue directions"),
+        null=True,
+        blank=True,
+        help_text=_("A link to the venue’s ‘How to find us’ page."),
```

Comment:
```suggestion
        help_text=_("A link to the venue's 'How to find us' page."),
```

Same here

Reply:
Replaced the apostrophe's with standard one. Don't why this happened in the first place.